docs/utils/mdHelper.js

```diff
@@ -28,13 +29,56 @@ export async function getDocByRoute(route: string): Promise<{|
   }
 }
 
-export async function getAllMarkdownPosts(): Promise<Array<string>> {
+export async function getAllMarkdownPosts(): Promise<Array<string[]>> {
+  function convertNamesForURL(name: string): string {
+    return name.replace(/ /g, '_').replace(/'/g, '').toLowerCase();
+  }
+
   const docsDirectory = path.join(process.cwd(), 'markdown');
 
-  const folderContents = await fs.readdir(docsDirectory);
-  const cleanedNames = folderContents
-    .filter((name) => name.endsWith('.mdx') || name.endsWith('.md'))
-    .map((name) => name.replace('.md', ''));
+  const getAllSitePaths = () => {
+    const pagePaths = [];
+
+    const addUrlPaths = (pageItems: Array<siteIndexType | string>, _pages: Array<string>) => {
+      const pages: Array<string> = _pages.concat([]);
+
+      // for each choice
+      pageItems.forEach((page) => {
+        if (page.sectionName) {
+          // $FlowFixMe[incompatible-type] This is a siteIndexType because it has a section name and isn't a string
```

Comment:
Can we fix these Flow issues?

Reply:
This is one is weird because our Site Index is a mixed type array of `string` or `SiteIndex`. Normally in typescript, I'd cast it. I didn't see a good way to do that in Flow, hence the $FixMe

Let me know if there's a better way to cast this correctly?